pkg/runner/cluster_k8s.go

```diff
@@ -337,8 +352,14 @@ func getPodLogs(clientset *kubernetes.Clientset, podName string) (string, error)
 	podLogOpts := v1.PodLogOptions{
 		TailLines: int64Ptr(2),
 	}
-	req := clientset.CoreV1().Pods("default").GetLogs(podName, &podLogOpts)
-	podLogs, err := req.Stream()
+
+	var podLogs io.ReadCloser
+	var err error
+	err = retry(20, 5*time.Second, func() error {
```

Comment:
I don't have good intuition on this retry, but I think in 100sec. we should get every container from Pending state to Running state, even for larger run beyond 1k testplan instances.